src/App.tsx

```diff
@@ -1,24 +1,247 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import {
+  FC,
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+import { addTodo, getTodos, removeTodo } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Loader } from './components/Loader';
+import { FilterBy } from './types/FilteredBy';
+import { TodoFilter } from './components/TodoFilter';
+import { Form } from './components/Form';
+import { ErrorMessage } from './components/ErrorMessage';
+import { setSingleOrPluralWordByCount } from './HelpersFunctions';
 
-const USER_ID = 0;
+const USER_ID = 6894;
 
-export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+function filterTodosByCompleted(todos: Todo[], filterBy: FilterBy): Todo[] {
+  switch (filterBy) {
+    case FilterBy.ACTIVE:
+      return todos.filter(todo => !todo.completed);
+
+    case FilterBy.COMPLETED:
+      return todos.filter(todo => todo.completed);
+
+    default:
+      return todos;
   }
+}
+
+export interface TodoItem {
+  todo: Todo,
+  isLoading: boolean,
+}
+
+export const App: FC = () => {
+  const [allTodos, setAllTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [todoTitle, setTodoTitle] = useState('');
+  const [filterType, setFilterType] = useState<FilterBy>(FilterBy.ALL);
+  const [tempTodos, setTempTodos] = useState<Todo | null>(null);
+  const [activeInput, setActiveInput] = useState(true);
+  const [isQuery, setIsQuery] = useState(false);
+  const [deletedTodoIds, setDeletedTodoIds] = useState<number[]>([]);
```

Comment:
Isn't this state used for showing loader? In that case rename it to `loadingTodoIds`